src/picongpu/include/plugins/output/GatherSlice.hpp

```diff
@@ -55,7 +62,9 @@ struct GatherSlice
      */
     bool init(bool isActive)
     {
-        /*free old communicator of init is called again*/
+        static int masterRankOffset = 0;
```

Comment:
yep, also cross-checked it with a minimal example. bit confusing that his works, but ok.